uber/templates/emails/reg_workflow/group_confirmation.html

```diff
@@ -17,7 +17,8 @@
     
     {% if not group.is_dealer %}
         <br/> <br/>
```

Comment:
Unrelated fix; we were hard-coding the value of ``5`` instead of using the config option.